widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/Merge.kt

```diff
@@ -31,11 +30,6 @@ internal inline infix fun <reified T : Any?> T.deepMerge(other: T): T {
     return unsafeMerge(other)
 }
 
-// Added to make method above compatible with Java
-internal fun deepMerge(old: UiTheme?, other: UiTheme?): UiTheme? {
```

Comment:
Just to double-check, `deepMerge()` is not needed anymore?

Reply:
Yep, it was created for one only Java method, which I converted to kotlin